administrator/components/com_associations/tmpl/autoassoc/modal.php

```diff
@@ -0,0 +1,255 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Session\Session;
+
+defined('_JEXEC') or die;
+
+HTMLHelper::_('jquery.framework');
+HTMLHelper::_('behavior.multiselect');
+HTMLHelper::_('script', 'com_associations/admin-autoassoc-modal.js', array('relative' => true));
+HTMLHelper::_('script', 'system/fields/modal-fields.min.js', array('version' => 'auto', 'relative' => true));
+
+$listOrder        = $this->escape($this->state->get('list.ordering'));
+$listDirn         = $this->escape($this->state->get('list.direction'));
+$canManageCheckin = JFactory::getUser()->authorise('core.manage', 'com_checkin');
+$colSpan          = 4;
+
+?>
+<button id="applyBtn" type="button" class="hidden" onclick="void(0);"></button>
+<button id="closeBtn" type="button" class="hidden" onclick="void(0);"></button>
+
+<div class="container-popup">
+	<form action="<?php echo JRoute::_('index.php?option=com_associations&view=autoassoc&layout=modal&id=' . $this->state->get('itemId')
+		. '&itemtype=' . $this->state->get('itemtype')
+	); ?>" method="post" name="adminForm" id="adminForm">
+		<div class="row">
+			<div class="col-md-12">
+				<div id="j-main-container" class="j-main-container">
+					<table class="table table-striped" id="languagesList">
+						<thead>
+						<tr>
+							<th style="width:1%" class="text-center">
+								<?php echo JHtml::_('grid.checkall'); ?>
+							</th>
+							<th style="width:5%" class="nowrap text-center">
+								<?php echo JText::_('JSTATUS'); ?>
+							</th>
+							<th style="width:10%" class="nowrap text-center">
+								<?php echo JText::_('JGRID_HEADING_LANGUAGE'); ?>
+							</th>
+							<th class="nowrap text-center">
+								<?php echo JText::_('JGLOBAL_TITLE'); ?>
+							</th>
+							<?php if (!empty($this->typeFields['catid'])) : ?>
+								<th class="nowrap text-center">
+									<?php echo JText::_('JCATEGORY'); ?>
+								</th>
+							<?php endif; ?>
+						</tr>
+						</thead>
+						<tfoot>
+						<tr>
+							<td colspan="<?php echo $colSpan; ?>">
+								<?php echo $this->pagination->getListFooter(); ?>
+							</td>
+						</tr>
+						</tfoot>
+						<tbody>
+						<?php foreach ($this->items as $i => $item) :?>
```

Comment:
Is it possible that $this->items is empty?

Reply:
This loop won't be executed if $this->items is false/null/an empty array. So I think it's ok.

Reply:
$this->items are the Content Languages.
If there are no Content Languages, there is no possible associations and, normally, the component will never display.